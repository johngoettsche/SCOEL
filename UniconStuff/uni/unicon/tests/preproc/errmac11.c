/* test error message: illegal redefinition of __TIME__ */

#define __TIME__(x) x
