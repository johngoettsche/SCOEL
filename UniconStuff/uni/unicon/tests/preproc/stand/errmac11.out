pp: File errmac11.c; Line 3: invalid redefinition of macro __TIME__
